Packs/MISP/Integrations/MISPV3/README.md

```diff
@@ -3934,27 +3934,27 @@ Search for attributes in MISP.
-| include_sightings | Whether to include the the sightings of the matching attributes. Default is false. Possible values: "true" and "false". Possible values are: true, false. | Optional | 
-| include_correlations | Whether to include the full correlations of the matching attributes. Possible values: "true" and "false". Default is false. Possible values are: true, false. | Optional | 
-| page | If a limit is set, sets the page to be returned. For example, page 3, limit 100 will return records 201-&gt;300. Default is 1. Default is 1. | Optional | 
-| limit | Limit the number of attributes returned. Default is 50. Default is 50. | Optional | 
-| enforceWarninglist | Whether to return only the values that are not on the warninglists. Possible values: "true" and "false". Possible values are: true, false. | Optional | 
-| compact | Whether to return only the attribute's values that match the search query. In case you want to get the full attributes data, set this argument to false. Possible values: "true" and "false". Possible values are: true, false. Default is false. | Optional | 
-
+| **Argument Name** | **Description**                                                                                                                                                                                                                                   | **Required** |
+| --- |---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| --- |
+| type | The attribute type. Use any valid MISP attribute type. For example: "md5", "sha1", "email", "url".                                                                                                                                                | Optional | 
+| value | Search for the specified value in the attribute's value field. For example: "1.2.3.4" (and other IP addresses), "google.com" (and other domains), "www.example.com" (and other URLs).                                                             | Optional | 
+| category | The attribute category. Use any valid MISP attribute category. For example: "Other", "Person", "Attribution", "Payload type".                                                                                                                     | Optional | 
+| uuid | Return attributes with the given UUID. Alternatively, return all the attributes that are part of the given UUID's event. For example, 59523300-4be8-4fa6-8867-0037ac110002.                                                                       | Optional | 
+| to_ids | Whether to return only the attributes set with the "to_ids" flag. The default is to return all attributes without with and with out to_ids flag. Possible values: "true" and "false". Possible values are: true, false.                           | Optional | 
```

Comment:
```suggestion
| to_ids | Whether to return only the attributes set with the "to_ids" flag. The default is to return all attributes without with and with out to_ids flag.  Possible values are: true, false.                           | Optional | 
```
There is something wrong with this sentence:
The default is to return all attributes without with and with out to_ids flag.